### PR TITLE
Point to Basho Webmachine 1.10.8

### PR DESCRIPTION
**Pull request by joedevivo**
Also updated to latest concrete

Our IPv6 changes have been merged upstream

### CODE REVIEW

**Comment by sdelano** [comment]
:+1: After this is merged please send out an email to dev notifying of the change.

I think the only further improvement that we may want is some way to track when new versions are tagged and released and be proactive about pulling them in.

File: rebar.config [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {deps,
     [{webmachine, ".*",
-       {git, "git://github.com/opscode/webmachine", {tag, "1.10.5.3"}}}
+       {git, "git://github.com/basho/webmachine", {tag, "1.10.8"}}}
```
Comment (marcparadise) [inline rebar.config]: Is this a case where we could use master?

**Comment by marcparadise** [comment]
:+1:

File: rebar.config (hunk 1 above)
Comment (joedevivo) [inline rebar.config]: we could, but maybe still err on the side of caution when pointing at external deps

**Pull request merged**